modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -20,6 +20,34 @@ Information on how to create a scheduler binary is outside the scope of this doc
 
 .Procedure
 
+. Create a file that contains a config map that holds the scheduler configuration file:
++
+.Example `scheduler-config-map.yaml`
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scheduler-config
+  namespace: kube-system <1>
+data:
+  scheduler-config.yaml: | <2>
+    apiVersion: kubescheduler.config.k8s.io/v1beta2
+    kind: KubeSchedulerConfiguration
+    profiles:
+      - schedulerName: custom-scheduler <3>
+----
+<1> This procedure uses the `kube-system` namespace, but you can use the namespace of your choosing.
+<2> When you define your `Deployment` resource later in this procedure, this file is passed in to the scheduler command by using the `--config` argument.
```

Comment:
"this file is passed" is passive voice. Maybe "The --config argument passes the file to the scheduler c command"? I don't know if that is true though.

Reply:
Yeah I had trouble with this one :smile: this one is a bit tricky, because I'm trying to say that you take this file and specify it later in the procedure for the --config arg. I'll take another stab at rewording using active voice though!